src/main/java/com/napier/sem/App.java

```diff
@@ -86,14 +88,14 @@ public void connect()
             System.exit(-1);
         }
 
-        int retries = 10;
+        int retries = 30;
         for (int i = 0; i < retries; ++i)
         {
             System.out.println("Connecting to database...");
             try
             {
                 // Wait a bit for db to start
-                Thread.sleep(30000);
+                Thread.sleep(10000);
```

Comment:
This just makes the app retry the database connection more regularly, reducing the average time you need to wait for a successful connection